### ISSUE TITLE
sidebar not present

### ISSUE THREAD

**Issue by vsreddyh** [KiKaraage/ArcWTF#84]
i followed all instructions exactly but when i imported the repo into chrome folder the sidebar vanished. the uc toggle isn't working. when i click the sideberry button the top tabs panel is switching where as the side tabs panel is nowhere to be found. i didn't have this issue previously before my system crash

**Comment by KiKaraage** [comment]
Yes, in the theme, the sidebar must be activated (press F1 to activate Sidebery) if you want the vertical bar to work - otherwise, if the sidebar is not active, it would shown the horizontal tab bar instead.

After the sidebar is activated, you can toggle the sidebar to be visible or not with the Userchrome Toggle button

Have you put the Userchrome Toggle configuration correctly?

**Comment by vsreddyh** [comment]
Yes userchrome is configured as you have mentioed in the readme file but F1 is not activating sidebery.

**Comment by KiKaraage** [comment]
Okay how about... Making a screen record to demo the not-working sidebar? With activating Sidebery, with clicking the UC Toggle button. Is the hovered sidebar working?

**Comment by vsreddyh** [comment]
Found the sidebar which is activating in hover mode after i toggled sidebery. however you can observe that the sidebey styles editor css is not being applied
[video(1).webm](https://github.com/user-attachments/assets/e22682dc-9225-49f8-900e-7920e8c7ed04)

**Comment by KiKaraage** [comment]
> sidebery styles editor css is not being applied

Yeah you have to copy-paste the code to the Style Editor in Sidebery settings page for that. I haven't find a workaround to pre-applied it thru scripts yet

**Comment by vsreddyh** [comment]
i already did. Another thing the styles applied(i could tell by the pinned tabs) before i pasted the zip contents in chrome folder. Also i customized the Navigation bar and it is there but not the styles editor
so current problems
1)UC toggle not working
2)Sidebery styles are not being applied

**Comment by mil1i** [comment]
Slightly different from original post, but close enough. Any ideas on what might be causing SideBerry to not be active on initially opening firefox?

Initial open I get the the tabs on top bar, and have to manually activate SideBerry for the sidebar to be present? After ensuring SideBerry is active; everything works as expected. Just need this to be active as default when opening firefox.

**Comment by vsreddyh** [comment]
I don't know. I moved on to zen-browser.

**Comment by geekloper** [comment]
I had the same issue, but I resolved it by adding a whitespace  after `|| ` prefix in the Userchrome Toggle config (as mentioned in the README, but, well, xD)
